db/migrate/20190912162820_create_subject_table.rb

```diff
@@ -8,73 +8,82 @@ def change
+        { subject_name: 'Primary with Geography and History', subject_code: '02' },
+        { subject_name: 'Primary with Mathematics', subject_code: '03' },
+        { subject_name: 'Primary with Modern Languages', subject_code: '04' },
+        { subject_name: 'Primary with Physical Education', subject_code: '06' },
+        { subject_name: 'Primary with Science', subject_code: '07' }
+      ]
 
-      secondary = ['Art and Design',
-                   'Science',
-                   'Biology',
-                   'Business Studies',
-                   'Chemistry',
-                   'Citizenship',
-                   'Classics',
-                   'Communication and Media Studies',
-                   'Computing',
-                   'Dance',
-                   'Design and Technology',
-                   'Drama',
-                   'Economics',
-                   'English',
-                   'Geography',
-                   'Health and Social Care',
-                   'History',
-                   'Mathematics',
-                   'Music',
-                   'Philosophy',
-                   'Physical Education',
-                   'Physics',
-                   'Psychology',
-                   'Religious Education',
-                   'Social Sciences'] + ['Modern Languages'] #no subject_code
+      secondary = [
+        { subject_name: 'Art and Design', subject_code: 'W1' },
+        { subject_name: 'Science', subject_code:  'F0' },
+        { subject_name: 'Biology', subject_code:  'C1' },
+        { subject_name: 'Business Studies', subject_code: '08' },
+        { subject_name: 'Chemistry', subject_code: 'F1' },
+        { subject_name: 'Citizenship', subject_code:  '09' },
+        { subject_name: 'Classics', subject_code: 'Q8' },
+        { subject_name: 'Communication and Media Studies', subject_code: 'P3' },
+        { subject_name: 'Computing', subject_code: '11' },
+        { subject_name: 'Dance', subject_code: '12' },
+        { subject_name: 'Design and Technology', subject_code: 'DT' },
+        { subject_name: 'Drama', subject_code: '13' },
+        { subject_name: 'Economics', subject_code: 'L1' },
+        { subject_name: 'English', subject_code: 'Q3' },
+        { subject_name: 'Geography', subject_code: 'F8' },
+        { subject_name: 'Health and Social Care', subject_code: 'L5' },
+        { subject_name: 'History', subject_code:  'V1' },
+        { subject_name: 'Mathematics', subject_code: 'G1' },
+        { subject_name: 'Music', subject_code: 'W3' },
+        { subject_name: 'Philosophy', subject_code: 'P1' },
+        { subject_name: 'Physical Education', subject_code: 'C6' },
+        { subject_name: 'Physics', subject_code: 'F3' },
+        { subject_name: 'Psychology', subject_code: 'C8' },
+        { subject_name: 'Religious Education', subject_code: 'V6' },
+        { subject_name: 'Social Sciences', subject_code: '14' },
+        #no subject_code
```

Comment:
suggest

    # no subject_code for Modern Languages because this is just a stub used to trigger
    # selection of actual entries from `modern_languages` list